app/pages/blackout/quote_section.jsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import devices from '../../styles/devices';
+
+import StartIcon from '../../../static/icons/quote_start.png';
+import EndIcon from '../../../static/icons/quote_end.png';
+
+const Wrapper = styled.section`
+    padding: 70px 0;
+`;
+
+const QuoteSection = styled.div`
+    position: relative;
+    width: 100%
+    margin: 0 auto;
+    padding: 30px 55px 60px 50px;
+    overflow: auto;
+  
+    ${devices.tablet`
+         padding: 30px 55px 60px 50px;
+         maxWidth: 75%;
+    `}
+`;
+
+const Quote = styled.div`
+    position: relative;
+    color: ${props => props.color};
+    float: ${props => (props.position === 'left' ? 'left' : 'right')};
+    zIndex: 1;
+    letter-spacing: 3px;
+    fontSize: 2.0em;
+    textAlign: left;
+
+    ${devices.tablet`
+        textAlign: ${props => (props.position === 'left' ? 'left' : 'right')};
+        minWidth: 30%;
+        width: 100%;
+        fontSize: 2.5em;
+        paddingBottom: 10px;
+    `}
+
+  ${devices.desktop`
+        textAlign: ${props => (props.position === 'left' ? 'left' : 'right')};
+        min-width: 30%;
+        width: 100%;
+        fontSize: 2.5em;
+  `}
+
+  ${devices.giant`
+        textAlign: ${props => (props.position === 'left' ? 'left' : 'right')};
+        min-width: 30%;
+        width: 80%;
+        fontSize: 2.5em;
+  `}
+`;
+
+const StartWrapper = styled.img`
+    position: absolute;
+    
+    height: 20%;
+    width: auto;
+    zIndex: 1;
+    visibility: hidden;
+    left: 0;
+    top: 0;
+    
+    ${devices.desktop`
+        height: 20%;
+        visibility: visible;
+    `}
+    
+    ${devices.tablet`
+        height: 20%;
+        visibility: visible;
+        
+    `}
+    
+    ${devices.giant`
+        height: 25%;
+        visibility: visible;
+    `}  
+`;
+
+const EndWrapper = styled.img`
+    position: absolute;
+    height: 20%;
+    width: auto;
+    visibility: hidden;
+    right: 0;
+    bottom: 4%;
+  
+  ${devices.desktop`
+        height: 20%;
+        visibility: visible;
```

Comment:
You don't need to repeat styles for higher size classes like visibility: visible for tablet, desktop, and giant, just having it for tablet will apply it to desktop and giant as well